apps/ejabberd/src/ejabberd.app.src

```diff
@@ -2,7 +2,7 @@
 
 {application, ejabberd,
  [{description, "ejabberd"},
-  {vsn, {cmd, "echo 2.1.8+mim-`git describe --always --tags`"}},
+  {vsn, {cmd, "echo 2.1.8-mim-`git describe --always --tags`"}},
```

Comment:
I'm absolutely sure it caused some strange error but I can't reproduce it. Maybe it got fixed by accident by one of the subsequent commits. :) I'll revert the change.